applications/ShallowWaterApplication/python_scripts/set_topography_process.py

```diff
@@ -0,0 +1,36 @@
+import KratosMultiphysics as KM
+import KratosMultiphysics.ShallowWaterApplication as SW
+
+def Factory(settings, Model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return SetTopographyProcess(Model, settings["Parameters"])
+
+## This process sets the value of a scalar variable using the AssignScalarVariableProcess.
+class SetTopographyProcess(KM.Process):
+
+    def __init__(self, Model, settings):
+
+        KM.Process.__init__(self)
+
+        default_settings = KM.Parameters("""
+            {
+                "mesh_id"              : 0,
```

Comment:
Do you need this?
I think it is considered legacy by now

Reply:
It is legacy. I was used in the no-submodelparts era.

Reply:
Removed